media/js/firefox/australis/fx36/whatsnew.js

```diff
@@ -0,0 +1,53 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    var $tourStrings = $('#strings');
+    var helloAnimationStage = $('#hello-animation-stage');
+
+    //track if this is the first time a user has seen tour
+    function trackFirstTimeUse() {
+        var firstTime = 'True';
+        try {
+            if (localStorage.getItem('mozUITourGlobalFlag') === 'taken') {
+                firstTime = 'False';
+            } else {
+                localStorage.setItem('mozUITourGlobalFlag', 'taken');
+            }
+            gaTrack(['_trackEvent', 'Tour Interaction', 'First Time Seeing Tour', firstTime, 0, true]);
```

Comment:
@alexgibson I agree that the majority of users will have seen a tour like this, but there will be some that have not. Since this data has been used in the past - and there could be some variance here - why not use it over a simple "View Page" event?